uktrade/data-hub-frontend
# Reactify investment project recipient company page

## Description of change

The page for assigning/editing a recipient company (AKA the UK company) for an investment project has been reactified using the same approach as #6034.

## Test instructions

Go to an investment project. It should be possible to set, edit and remove a recipient company.

## Screenshots

![Screenshot 2023-08-31 at 11-29-26 Add recipient company - Team page project - Projects - Investments - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/d162f110-cab6-4087-9321-c6c52d015beb)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
